examples/k8s/dex.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: dex
+  name: dex
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: dex
+    spec:
+      containers:
+      - image: quay.io/coreos/dex:v2.0.0-beta.1
+        name: dex
+        command: ["/usr/local/bin/dex", "serve", "/etc/dex/cfg/config.yaml"]
+
+        ports:
+        - containerPort: 5556
+          name: https
```

Comment:
total style nit: I've started putting `name` first if an object supports it (ports, metadata, container), not sure if that matters to ya. Its inconsistent atm.